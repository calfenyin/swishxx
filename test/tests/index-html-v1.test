index | | -e html:*.html -v1 | html | 0
