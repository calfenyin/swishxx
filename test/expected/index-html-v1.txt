
index: done:
  5 indexed
  2448 words, 1295 indexed, 496 unique

